### PR TITLE
Upgrade GitLab to 9.4.2

### PR DESCRIPTION
**Pull request by solidnerd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1197,20 +1197,20 @@ Usage when using `docker-compose` can also be found there.
 >
 > Since GitLab release `8.6.0` PostgreSQL users should enable `pg_trgm` extension on the GitLab database. Refer to GitLab's [Postgresql Requirements](http://doc.gitlab.com/ce/install/requirements.html#postgresql-requirements) for more information
 >
-> If you're using `sameersbn/postgresql` then please upgrade to `sameersbn/postgresql:9.4-18` or later and add `DB_EXTENSION=pg_trgm` to the environment of the PostgreSQL container (see: https://github.com/sameersbn/docker-gitlab/blob/master/docker-compose.yml#L8).
+> If you're using `sameersbn/postgresql` then please upgrade to `sameersbn/postgresql:9.4.28` or later and add `DB_EXTENSION=pg_trgm` to the environment of the PostgreSQL container (see: https://github.com/sameersbn/docker-gitlab/blob/master/docker-compose.yml#L8).
```
Comment (nekozanmai) [inline README.md]: I think that 'sameersbn / postgresql: 9.4.28' is wrong.

**Comment by abeltay** [comment]
Remember to update the changelog

File: README.md (hunk 1 above)
Comment (solidnerd) [inline README.md]: Thanks for pointing that out.

**Comment by solidnerd** [comment]
LGTM !

**Pull request merged**